docs/articles/documentation/test-api/intercepting-http-requests/mocking-http-responses.md

```diff
@@ -117,3 +119,18 @@ Property | Type | Description
 Method | Description
 ------ | ---------------
 `setBody(value)` | Sets the response body.  Accepts a string as a parameter.
+
+## Examples
+
+### Mocking Cross-Domain Responses
+
+When you mock a cross-domain response (a response to a request to a different domain), set the response's `access-control-allow-origin` header to the tested site's domain (or an asterisk `*`) to pass [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) validation.
```

Comment:
`When you mock a cross-domain requests, ...`.

Cross-domain request is a common term. You don't need explain this.

There are two most popular cases:
* if 'access-control-allow-credentials' is specified, then you should  setup an exact domain for 'access-control-allow-origin' header. With `*` specified, the CORS will fail in this case.
* if 'access-control-allow-credentials' is not specified, then you can specify the exact domain or `*`.